src/util/locking/RedisResourceLocker.ts

```diff
@@ -1,6 +1,6 @@
 import { assert } from 'console';
-import type { RedisClient } from 'redis';
```

Comment:
@joachimvh I think we should ignore the warning below; up to you whether locally or globally.

Reply:
Yes I was planning to look into this to see if we need to change the linter settings or the import later. Was actually surprised our linter could throw warnings so also going to change that so it's always errors.